lib/travis/build/script/r.rb

```diff
@@ -112,12 +112,13 @@ def configure
                   'cdbs qpdf texinfo libssh2-1-dev devscripts '\
                   "#{optional_apt_pkgs}", retry: true
 
-                r_filename = "R-#{r_version}-$(lsb_release -cs).xz"
-                r_url = "https://travis-ci.rstudio.org/#{r_filename}"
+                r_filename = "r-#{r_version}_1_amd64.deb"
```

Comment:
Is R meant to work on ARM, or just x86_64? If the former, the file needs to be appropriately specified, and if the latter, the build just halt.

Reply:
Just x86_64, what is the best way to condition on ARM?

Reply:
I think this is address in the `if` above.